schema/deploy/computed_columns/form_change_holdback_amount_to_date.sql

```diff
@@ -7,15 +7,26 @@ create or replace function cif.form_change_holdback_amount_to_date(parameter_fc
 returns numeric as
 $fn$
 
-  select
-    round(sum(coalesce((fc.new_form_data->>'adjustedHoldbackAmount')::numeric, (fc.new_form_data->>'calculatedHoldbackAmount')::numeric, 0)), 2)
+  with results as (
+    select
+        (fc.new_form_data->>'adjustedHoldbackAmount')::numeric as adjustedHoldbackAmount,
+        (fc.new_form_data->>'calculatedHoldbackAmount')::numeric as calculatedHoldbackAmount
     from cif.form_change fc
     where fc.project_revision_id = $1.project_revision_id
     and json_schema_name = 'milestone'
-    and (fc.new_form_data->>'hasExpenses')::boolean = true;
+    and (fc.new_form_data->>'hasExpenses')::boolean = true
+    and fc.operation <> 'archive'
+  )
+
+  select
+    case
+      when count(*) filter (where adjustedHoldbackAmount is null and calculatedHoldbackAmount is null) > 0 then null
```

Comment:
is `and` the correct condition here? If someone's filled out the adjusted holdback amount then I think they do have enough info to do the calculation

Reply:
This filter creates a condition where if any milestone is missing both values then the amount_to_date cannot be calculated